plugins/inputs/odyssey/odyssey.go

```diff
@@ -0,0 +1,158 @@
+package odyssey
+
+import (
+	"bytes"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/plugins/inputs/postgresql"
+	_ "github.com/jackc/pgx/stdlib" // register driver
+)
+
+type Odyssey struct {
+	postgresql.Service
+	IncludeQuery []string `toml:"include_query"`
+}
+
+var ignoredColumns = map[string]bool{"user": true, "database": true, "pool_mode": true,
+	"avg_req": true, "avg_recv": true, "avg_sent": true, "avg_query": true,
+}
+
+var sampleConfig = `
+  ## specify address via a url matching:
+  ##   postgres://[console[:password]]@localhost[/dbname]\
+  ##       ?sslmode=[disable|verify-ca|verify-full]
+  ## or a simple string:
+  ##   host=localhost user=console password=... sslmode=... dbname=app_production
+  ##
+  ## include_query = []string - commands for psql
+  ##
+  ## All connection parameters are optional.
+  ##
+  address = "host=localhost user=postgresql sslmode=disable dbname=console port=6432"
+
+  include_query = ['SHOW STATS', 'SHOW POOLS']
```

Comment:
Hmmm... You require the user to specify an `include_query` as otherwise you will not collect any metric. How about making the specified queries the default ones (i.e. use those if no include_query is given) or at least check (e.g. in an `Init()` function) if the parameter is empty?
It would also help to move the parameter description directly above the parameter instead of having a common description section. You might want to specify more possible values here...

Reply:
agreed